### ISSUE TITLE
Clarify usage of aria-haspopup and aria-expanded for elements that trigger dialogs

### ISSUE THREAD

**Issue by 0ddfell0w** [w3c/aria-practices#1130]
Could we add a mention of the states and properties expected on elements that trigger dialogs? Perhaps as a "Note" on the relevant pattern(s)

For the Dialog and Tooltip Dialog (#128) patterns, we could suggest
1. the element that triggers a dialog should have aria-haspopup=dialog. 
2. the element that triggers a dialog should use aria-expanded to reflect whether the dialog is showing
